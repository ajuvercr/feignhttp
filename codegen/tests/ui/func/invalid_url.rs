use feignhttp_codegen::get;

// error: metadata url is invalid
//    |  #[get(url + "/aaa")]
//    |  ^^^^^^^^^^^^^^^^^^^^

const url: &str = "http://xxx";

#[get(url + "/aaa")]
fn send_get() {}

fn main() {}
