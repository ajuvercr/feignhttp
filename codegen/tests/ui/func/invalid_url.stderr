error: metadata url is invalid
 --> tests/ui/func/invalid_url.rs:9:1
  |
9 | #[get(url + "/aaa")]
  | ^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `get` (in Nightly builds, run with -Z macro-backtrace for more info)
